back-end/gateway/src/main/java/org/functionstream/gateway/common/BusAddressConst.java

```diff
@@ -0,0 +1,14 @@
+package org.functionstream.gateway.common;
+
+/**
+ * @author HALOXIAO
```

Comment:
According to 'Code Complete', comments are used to illustrate the purpose of the code. Using it for other purposes like specifying the ownership may result in 'comment decay'.
That said, keeping track of code ownership, change-log and who last modify the file, is the job of a source control tool like git, and should not be inside the comments.

I think we need to add some comments here to Illustrate what this interface is for.